solution/ui/regulations/js/src/components/RelatedRule.vue

```diff
@@ -11,7 +11,8 @@
                     v-if="type !== ''"
                     class="recent-flag indicator"
                     :class="indicatorClasses"
-                >{{type}}</span>
+                    >{{ type }}</span
+                >
```

Comment:
Is this the closing tag for </span placed on the next line?

Reply:
indeed.  I use prettier to format code via a hotkey and it has some interesting opinions on how to format tags like this.  The real reason is that it ensures that no empty spaces are included on either side of the `type` variable, which can happen when you have the span tags on different lines.  I liked it how it was before this change, but I also don't mind prettier's change.  This is something we can add to the list of things we'd like to adjust with our linters and formatters.

Reply:
Agreed!